src/c2goto/library/pthread_lib.c

```diff
@@ -483,23 +582,90 @@ __ESBMC_HIDE:;
   return 0;
 }
 
+// The pthread_key_create() function shall create a thread-specific
+// data key visible to all threads in the process.
+// source: https://linux.die.net/man/3/pthread_key_create
```

Comment:
C-style comments.

Reply:
I have followed the style of the comments in ESBMC.